doc: nrf5340audio: add docs for new apps (#37)
alexsven/sdk-nrf: applications/nrf5340_audio/doc/user_interface.rst

| **DEBUG ENABLE**  | Turns on or off power for debug features.                                           | All                                   |
|                   | This switch is used for accurate power and current measurements.                    |                                       |
+-------------------+-------------------------------------------------------------------------------------+---------------------------------------+

.. _nrf53_audio_app_ui_buttons:

Buttons
*******

The application uses the following buttons on the supported development kit:

+---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
| Button        | Function                                                                                                  | Applications                                |
+===============+===========================================================================================================+=============================================+

> | **VOL-**      | Long-pressed during startup: Changes the headset to the left channel one.                                 | All                                         |

Reviewer: Only valid for unicast_server and broadcast_sink, same for VOL+

|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
|               | Pressed on a BIS headset during playback: Change audio stream, if more than one is available.             | :ref:`nrf53_audio_broadcast_sink_app`       |
+---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
| **BTN 5**     | Long-pressed during startup: Clears the previously stored bonding information.                            | * :ref:`nrf53_audio_unicast_server_app`     |
|               |                                                                                                           | * :ref:`nrf53_audio_unicast_client_app`     |
|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+

> |               | Pressed during playback: Mutes the playback volume.                                                       | All                                         |

Reviewer: ```suggestion
|               | Pressed during playback: Mutes the playback volume.                                                       | * :ref:`nrf53_audio_unicast_server_app`     |
|               |                                                                                                           | * :ref:`nrf53_audio_unicast_client_app`                                         |
```

Reviewer: Formatting is probably not correct, but it is like I stated previously, only the unicast applications will use BTN5 to mute

|                          |                                                                                                           | * :ref:`nrf53_audio_unicast_server_app`     |
|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
|                          | Blinking blue on the CIS gateway: Kit has connected to a headset.                                         | :ref:`nrf53_audio_unicast_client_app`       |
|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
|                          | Blinking blue on the BIS gateway: Kit has started broadcasting audio.                                     | :ref:`nrf53_audio_broadcast_source_app`     |
|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+

> |                          | Solid blue on the CIS headset: Kits have connected to the gateway.                                        | :ref:`nrf53_audio_unicast_server_app`       |

Reviewer: This is true for both headset and gateway, so maybe change it to:
```suggestion
|                          | Solid blue on the CIS gateway and headset: Kits have connected.                                        | :ref:`nrf53_audio_unicast_server_app`   :ref:`nrf53_audio_unicast_client_app`    |
```

|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+

> |                          | Solid blue on the BIS headset: Kits have found a broadcasting stream.                                     | :ref:`nrf53_audio_broadcast_sink_app`       |

Reviewer: I also think we should do solid first, then blinking, since that is the order they will appear when using the app

| **BTN 4**     | Long-pressed during startup: Turns on the DFU mode, if                                                    | All                                         |
|               | the device is :ref:`configured for it<nrf53_audio_app_configuration_configure_fota>`.                     |                                             |
|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
|               | Pressed on the gateway during playback: Toggles between the normal audio stream and different test        | * :ref:`nrf53_audio_broadcast_source_app`   |
|               | tones generated on the device. Use this tone to check the synchronization of headsets.                    | * :ref:`nrf53_audio_unicast_client_app`     |
|               +-----------------------------------------------------------------------------------------------------------+                                             |

> |               | Pressed on the gateway during playback multiple times: Changes the tone frequency.                        |                                             |

Reviewer 2: ```suggestion
|               | Pressed on the gateway during playback multiple times: Changes the test tone frequency.                        |                                             |
```

|               | The available values are 1000 Hz, 2000 Hz, and 4000 Hz.                                                   |                                             |
|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
|               | Pressed on a BIS headset during playback: Change audio stream, if more than one is available.             | :ref:`nrf53_audio_broadcast_sink_app`       |
+---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
| **BTN 5**     | Long-pressed during startup: Clears the previously stored bonding information.                            | All                                         |
|               +-----------------------------------------------------------------------------------------------------------+                                             |

> |               | Pressed during playback: Mutes the playback volume.                                                       |                                             |

Reviewer: It's not muting for all devices, since broadcast sink will change gateway when BTN5 is pressed and broadcast source doesn't use BTN5

Reviewer: These should be listed as muting
* :ref:`nrf53_audio_unicast_server_app` ,
* :ref:`nrf53_audio_unicast_client_app`

+==========================+===========================================================================================================+=============================================+
| **LED1**                 | Off - No Bluetooth connection.                                                                            | All                                         |
|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
|                          | Blinking blue on headset: Kits have started streaming audio (BIS and CIS modes).                          | * :ref:`nrf53_audio_broadcast_sink_app`     |
|                          |                                                                                                           | * :ref:`nrf53_audio_unicast_server_app`     |
|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+

> |                          | Blinking blue on the CIS gateway: Kit has connected to a headset.                                         | :ref:`nrf53_audio_unicast_client_app`       |

Reviewer: ```suggestion
|                          | Blinking blue on the CIS gateway: Kit is streaming audio to a headset.                                         | :ref:`nrf53_audio_unicast_client_app`       |
```

Reviewer: Not fixed
